charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
```

```suggestion
    gardener.cloud/role: controlplane
```
This is new resource, and I think that we can use the new label key. @rfranzke wdyt?

---

Yes, it is okay to use the new label, thank you. The `garden.sapcloud.io/role` label is just use for workload resources like `Deployment`, etc., in order to check the health of the running pods. This is a CR, so it's okay to use the new label as it won't be evaluated anyways.